yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,68 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		}),
 	}
 
-	unsupportedDatatypesList := lo.Map(assessmentReport.UnsupportedDataTypes, func(datatype utils.TableColumnsDataTypes, _ int) string {
-		return datatype.DataType
-	})
+	explanation, err := buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		log.Errorf("failed to build migration complexity explanation: %v", err)
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:              issue.Category,
+			CategoryDescription:   issue.CategoryDescription,
+			Type:                  issue.Type,
+			Name:                  issue.Name,
+			Description:           issue.Description,
+			Impact:                issue.Impact,
+			ObjectType:            issue.ObjectType,
+			ObjectName:            constants.OBFUSCATE_STRING,
+			SqlStatement:          constants.OBFUSCATE_STRING, // TODO(future): we can obfuscate sensitive info in SQL statement
+			DocsLink:              issue.DocsLink,
+			MinimumVersionFixedIn: issue.MinimumVersionFixedIn,
+		}
 
-	groupedByConstructType := lo.GroupBy(assessmentReport.UnsupportedQueryConstructs, func(q utils.UnsupportedQueryConstruct) string {
-		return q.ConstructTypeName
-	})
-	countByConstructType := lo.MapValues(groupedByConstructType, func(constructs []utils.UnsupportedQueryConstruct, _ string) int {
-		return len(constructs)
-	})
+		// allowing object name for unsupported extension issue type
+		if issue.Type == UNSUPPORTED_EXTENSION_ISSUE_TYPE {
+			// object name(i.e. extension name here) might be qualified with schema so just taking the last part
+			obfuscatedIssue.ObjectName = strings.Split(issue.ObjectName, ".")[len(strings.Split(issue.ObjectName, "."))-1]
```

Comment:
generally extension name is not qualified as the CREATE EXTENSION DDL has a separate option for mentioning the schema name and I believe we are not using the schema name. 
```
CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;
```